### ISSUE TITLE
Accessibility testing v3.0.11: Notification bell colours 

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#566]
The colour of the notification bell in dark mode should be changed: 

- The background of text colour to #0078D7
- Notification icon itself colour to #F4F7FA
- And the text colour to #F4F7FA as well

**Comment by weglmar** [comment]
Seems to be resolved on my end (colours look good). 
@Rajat-Sen, could you please confirm?

**Comment by Rajat-Sen** [comment]
@weglmar, I cannot see the text colour on the bell icon yet. However, the bell icon itself is resolved.

**Comment by Rajat-Sen** [comment]
Validated with the text colour icon. Resolved.
